### ISSUE TITLE
Pie chart/donut chart

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#1672]
### Description: What is it?

I want to be able to build a pie chart/donut chart. Where the radians = the proportion of the total value that each group is.

### Problem: What problem does this solve?

Sometimes, I want to show my data as a pie chart

### What: Roughly, what does this look like in the product?

an option in the chart configs for a donut/pie chart. 

This option can only be applied to a two field visualization.

**Comment by hamzahc1** [comment]
https://echarts.apache.org/examples/en/editor.html?c=pie-doughnut 🥳 🍩

**Comment by hamzahc1** [comment]
Can we definitely make these donut charts, not pie charts 🤮

**Comment by rephus** [comment]
I'm looking into this, it sounds like a big task.I'll see what I can do

**Comment by TuringLovesDeathMetal** [comment]
@rephus if it's a big task, then it's not a massive priority - so I'd stick it a bit lower in the list of things to do! 

(if it will take more than a full day of work - then let's de-prioritize it :) )

**Comment by rephus** [comment]
I got 1 implentation question. Should this be a new `ChartType` ? (like Table or BigNumber) or be type of `Cartesian` chart (CartesianSeriesType) ...

**Comment by TuringLovesDeathMetal** [comment]
New `ChartType`

**Comment by rephus** [comment]
I can spend some time to figure out how difficult this is. But if you have something else more urgent, let me know.

**Comment by rephus** [comment]
Yeah, I made a draft a while ago but we deprioritized this, not sure how relevant this is anymore @TuringLovesDeathMetal

**Comment by TuringLovesDeathMetal** [comment]
ya, it is...just not super high priority. 

We will forever be building more chart types 😂
